lax/lichens/sciencerun0.py

```diff
@@ -623,6 +625,48 @@ def _process(self, df):
             raise ValueError('Only versions 2 and 3 are implemented')
 
 
+class CS2AreaFractionTop(ManyLichen):
+    """cS2 area fraction top cut
+
+    Designed to target gas events.
+    The acceptance is 99% by design.
+    Valid in S2 range 0-10000
+
+    See the note at xenon:xenon1t:adam:s2aft:sr1_cs2_cut
+    """
+    version = 0
+
+    class CS2AreaFractionTopUpper(StringLichen):
+        """cS2 AFT upper bound
+        """
+        string = 'cs2_aft < 0.63756073 + 1.42873942 / sqrt(s2)'
+
+    class CS2AreaFractionTopLower(StringLichen):
```

Comment:
I would argue for a looser acceptance on the lower part. upper part is used against gas events we are most afraid of, so we can tolerate 1-2% acceptance loss. The lower one can be modified so acceptance is almost unity.